tasks.py

```diff
@@ -34,23 +41,24 @@ def is_truthy(arg):
 
 # Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
 # Variables may be overwritten in invoke.yml or by the environment variables INVOKE_NAUTOBOT_CHATOPS_xxx
-
 namespace = Collection("nautobot_chatops")
 namespace.configure(
     {
         "nautobot_chatops": {
-            "nautobot_ver": "1.4.0",
+            "nautobot_ver": "1.5.24",
```

Comment:
This should match the min version of 1.5.4

Reply:
Fixed